### PR TITLE
create multus kubeconfig for in case of non auto flag.

### LINKED ISSUE #1295: thin_entrypoint should generate multus kubeconfig with manual option
Older version of multus e.g. 3.9.3 with older script of thin_entry.sh generates a multus.d directory and multus.kubeconfig file with a non-auto installation e.g. # "--multus-conf-file=/tmp/multus-conf/70-multus.conf"

The new version 4.0.2 with new thin_entrypoint doesn't generate multus.d and multus.kubeconfig if the installation type is non-auto. We still want multus to generate multus.kubeconfig even though if the configfile is used manually.

I believe adding below lines [here ](https://github.com/k8snetworkplumbingwg/multus-cni/blob/9f5c0239a804a90bf8913c9a386ca50eff9ce0cc/cmd/thin_entrypoint/main.go#L584)would fix it. 
`caHash, saTokenHash, err = opt.createKubeConfig(nil, nil)
		if err != nil {
			fmt.Fprintf(os.Stderr, "failed to create multus kubeconfig: %v\n", err)
			return
		}`

### PR DESCRIPTION
**Pull request by adrianchiris**
This is a small rework of PR#1302 to reduce code changes for test.

Fixes #1295

### CODE REVIEW

**Comment by adrianchiris** [comment]
This PR is based on #1302 , after rebase on master and some streamlining of the test to reduce code changes in thin_entrypoint.

**Comment by adrianchiris** [comment]
@dougbtv PTAL

File: cmd/thin_entrypoint/main_test.go [hunk 1]
```diff
@@ -5,12 +5,35 @@ package main
...
+	"path/filepath"
+	"syscall"
 	"testing"
 
 	. "github.com/onsi/ginkgo/v2" //nolint:golint
 	. "github.com/onsi/gomega"    //nolint:golint
 )
 
+// chrootTestHelper performs chroot syscall, returns func to get back to original root or error if occurred
+func chrootTestHelper(path string) (func() error, error) {
```
Comment (dougbtv) [inline cmd/thin_entrypoint/main_test.go:17]: smart!

File: cmd/thin_entrypoint/main.go [hunk 2]
```diff
@@ -585,6 +585,11 @@ func main() {
 	var masterConfigFilePath string
 	// copy user specified multus conf to CNI conf directory
 	if opt.MultusConfFile != "auto" {
+		caHash, saTokenHash, err = opt.createKubeConfig(nil, nil)
```
Comment (dougbtv) [inline cmd/thin_entrypoint/main.go:588]: nice and efficient.

**Review by dougbtv — APPROVED** [review]
thanks for the rework Adrian!

**Pull request merged**